plugins/rbac/README.md

```diff
@@ -21,14 +21,28 @@ Follow the RBAC backend plugin [README](https://github.com/janus-idp/backstage-p
 
 **NOTE**
 
-To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
+- To enable create role button on Administration -> RBAC roles list page, the role associated with your user should have the following permission policies associated with it. Add the following in your permission policies configuration file:
 
 ```CSV
-p, role:default/team_a, catalog-entity, read, allow
+p, role:default/team_a, catalog-entity, read, allow  # only needed for users with the role other than `role:default/rbac_admin`
 p, role:default/team_a, policy-entity, create, allow
 g, user:default/<login-id/user-name>, role:default/team_a
```

Comment:
this looks a bit confusing based on Readme if you add a user in config https://github.com/janus-idp/backstage-plugins/pull/1155/files#diff-8ede316b0bcf371c5e1b99eebe533e18667172539b0492c668976d14af9e9483R43 then they automatically will be mapped to `role:default/rbac_admin` and they will have access to so should we have this note?

https://github.com/janus-idp/backstage-plugins/pull/1137 handled this

@AndrienkoAleksandr WDYT?

Reply:
I have updated this to mention non-admin users